session/manager.go

```diff
@@ -74,6 +74,30 @@ func (sm *Manager) SetLocale(ctx context.Context, locale string) error {
 	return err
 }
 
+// SetSessionTimeout changes vCenter's default session timeout (in minutes).
+// Changes HostAgent's session timeout when connected to an ESX host.
+// Note that vcsim accepts a time.Duration.String(),
+// but vCenter+ESX only accept an integer string, in minutes.
+func (sm *Manager) SetSessionTimeout(ctx context.Context, minutes string) error {
```

Comment:
I'm wondering if this is the proper interface. I understand that internally config uses strings for minutes but I'm wondering if perhaps `SetSessionTimeout` should take `Duration` and internally convert it to `string`?

Reply:
I considered doing that, problem is real VC can't parse fractions, so we'd have to round up to the nearest whole second. Don't really want to add a client-side check there to see if we're talking to vcsim. Thinking I'll move this helper to the simulator package and change it to use `time.Duration`, making it clear in the doc it's only meant for use with vcsim. It'll still be a handy 1-liner for tests and example for porting to other languages using vcsim. What do you think?

```console
% govc option.set config.vmacore.soap.sessionTimeout .250 # against real VC
govc: ServerFaultCode: A general system error occurred: Config Value not an integer
```

Reply:
Sounds good to me. I like the idea of having this function specialized for vcsim.

Reply:
ok, done. Created a new `simulator/sim25` package, have some other helpers in mind that could live there.